### PR TITLE
add new expired status

### PR DESCRIPTION
**Pull request by epenameli**
## Description
In few word, describe what’s your Pull Request about.

### Issue involved
Fixes #<issue>

### Checklist:
Before sending your contribution, please specify the following: 

- [ ] This request modifies code snippets. 
- [X] The contribution aligns with the information stated in the repository wiki.

- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [ ] Contribution includes new features. 
- [ ] New features were communicated in changelog.
- [X] Requires translations.

Se añadió el nuevo status expired a la documentación de merchant orders. Necesita traducción.
Se añadió el campo date of expiration a la api de preferencias. Necesita traducción.

### CODE REVIEW

File: reference/preferences/resource.yaml [hunk 1]
```diff
@@ -418,6 +418,12 @@ properties:
       en: Boolean value that determines if a preference expire.
       es: Preferencia que determina si una preferencia expira.
       pt: Preferência que determina se uma preferência expira.
+- date_of_expiration:
+    type: Date(ISO_8601)
+    description:
+      en: 
+      es: Fecha de expiración del medio de pago en  efectivo.
```
Comment (pamalv7) [inline reference/preferences/resource.yaml:425]:
```suggestion
      es: Fecha de vencimiento del medio de pago en efectivo.
```

File: reference/preferences/resource.yaml [hunk 2]
```diff
@@ -418,6 +418,12 @@ properties:
       en: Boolean value that determines if a preference expire.
       es: Preferencia que determina si una preferencia expira.
       pt: Preferência que determina se uma preferência expira.
+- date_of_expiration:
+    type: Date(ISO_8601)
+    description:
+      en:
```
Comment (pamalv7) [inline reference/preferences/resource.yaml:424]:
```suggestion
      en: Expiration date of cash payment.
```

File: reference/preferences/resource.yaml [hunk 3]
```diff
@@ -418,6 +418,12 @@ properties:
       en: Boolean value that determines if a preference expire.
       es: Preferencia que determina si una preferencia expira.
       pt: Preferência que determina se uma preferência expira.
+- date_of_expiration:
+    type: Date(ISO_8601)
+    description:
+      en: 
+      es: Fecha de expiración del medio de pago en  efectivo.
+      pt:
```
Comment (pamalv7) [inline reference/preferences/resource.yaml:426]:
```suggestion
      pt: Data de expiração de meios de pagamento em dinheiro.
```

File: reference/merchant_orders/resource.yaml [hunk 4]
```diff
@@ -46,6 +46,10 @@ properties:
         en: Order with payments covering total amount.
         es: Orden con pagos que cubren el monto total.
         pt: Order with payments covering total amount.
+    - expired:
+        en:
```
Comment (pamalv7) [inline reference/merchant_orders/resource.yaml:50]:
```suggestion
        en: Canceled order (canceled: true) that does not have approved or pending payments (all rejected or returned).
```

File: reference/merchant_orders/resource.yaml [hunk 5]
```diff
@@ -46,6 +46,10 @@ properties:
         en: Order with payments covering total amount.
         es: Orden con pagos que cubren el monto total.
         pt: Order with payments covering total amount.
+    - expired:
+        en: 
+        es: Orden cancelada (cancelled:true) que no posee pagos aprobados ni pendientes (todos rechazados o devueltos).
+        pt:
```
Comment (pamalv7) [inline reference/merchant_orders/resource.yaml:52]:
```suggestion
        pt: Pedido cancelado (cancelado: verdadeiro) que não possui pagamentos aprovados ou pendentes (todos rejeitados ou devolvidos).
```

Comment (pamalv7) [reply reference/merchant_orders/resource.yaml:52]: @lucaskewin, ¿te parece bien esto?

File: reference/merchant_orders/resource.yaml [hunk 6]
```diff
@@ -801,4 +805,8 @@ properties:
     - undefined:
         en: Order created before the existence of the order_status status.
         es: Orden creada antes de la existencia del estado order_status.
-        pt: Order created before the existence of the order_status status.
\ No newline at end of file
+        pt: Order created before the existence of the order_status status.
+    - expired:
+        en:
```
Comment (pamalv7) [inline reference/merchant_orders/resource.yaml:810]:
```suggestion
        en: Canceled order (canceled: true) that does not have approved or pending payments (all rejected or returned).
```

File: reference/merchant_orders/resource.yaml [hunk 7]
```diff
@@ -801,4 +805,8 @@ properties:
     - undefined:
         en: Order created before the existence of the order_status status.
         es: Orden creada antes de la existencia del estado order_status.
-        pt: Order created before the existence of the order_status status.
\ No newline at end of file
+        pt: Order created before the existence of the order_status status.
+    - expired:
+        en: 
+        es: Orden cancelada (cancelled:true) que no posee pagos aprobados ni pendientes (todos rechazados o devueltos)
+        pt:
```
Comment (pamalv7) [inline reference/merchant_orders/resource.yaml:812]:
```suggestion
        pt: Pedido cancelado (cancelado: verdadeiro) que não possui pagamentos aprovados ou pendentes (todos rejeitados ou devolvidos).
```

File: reference/merchant_orders/resource.yaml [hunk 8]
```diff
@@ -46,6 +46,10 @@ properties:
         en: Order with payments covering total amount.
         es: Orden con pagos que cubren el monto total.
         pt: Order with payments covering total amount.
+    - expired:
+        en: Canceled order (canceled: true) that does not have approved or pending payments (all rejected or returned).
+        es: Orden cancelada (cancelled:true) que no posee pagos aprobados ni pendientes (todos rechazados o devueltos).
+        pt: Pedido cancelado (cancelado: verdadeiro) que não possui pagamentos aprovados ou pendentes (todos rejeitados ou devolvidos).
```
Comment (lamaralmp) [inline reference/merchant_orders/resource.yaml:52]:
```suggestion
        pt: Pedido cancelado (cancelled: true) que não possui pagamentos aprovados ou pendentes (todos rejeitados ou devolvidos).
```

File: reference/merchant_orders/resource.yaml [hunk 9]
```diff
@@ -46,6 +46,10 @@ properties:
         en: Order with payments covering total amount.
         es: Orden con pagos que cubren el monto total.
         pt: Order with payments covering total amount.
+    - expired:
+        en: Canceled order (canceled: true) that does not have approved or pending payments (all rejected or returned).
+        es: Orden cancelada (cancelled:true) que no posee pagos aprobados ni pendientes (todos rechazados o devueltos).
```
Comment (lamaralmp) [inline reference/merchant_orders/resource.yaml:51]:
```suggestion
        es: Orden cancelada (cancelled: true) que no posee pagos aprobados ni pendientes (todos rechazados o devueltos).
```

File: reference/merchant_orders/resource.yaml [hunk 10]
```diff
@@ -801,4 +805,8 @@ properties:
     - undefined:
         en: Order created before the existence of the order_status status.
         es: Orden creada antes de la existencia del estado order_status.
-        pt: Order created before the existence of the order_status status.
\ No newline at end of file
+        pt: Order created before the existence of the order_status status.
+    - expired:
+        en: Canceled order (canceled: true) that does not have approved or pending payments (all rejected or returned).
+        es: Orden cancelada (cancelled:true) que no posee pagos aprobados ni pendientes (todos rechazados o devueltos)
+        pt: Pedido cancelado (cancelado: verdadeiro) que não possui pagamentos aprovados ou pendentes (todos rejeitados ou devolvidos).
```
Comment (lamaralmp) [inline reference/merchant_orders/resource.yaml:812]:
```suggestion
        pt: Pedido cancelado (cancelled: true) que não possui pagamentos aprovados ou pendentes (todos rejeitados ou devolvidos).
```

File: reference/merchant_orders/resource.yaml [hunk 11]
```diff
@@ -801,4 +805,8 @@ properties:
     - undefined:
         en: Order created before the existence of the order_status status.
         es: Orden creada antes de la existencia del estado order_status.
-        pt: Order created before the existence of the order_status status.
\ No newline at end of file
+        pt: Order created before the existence of the order_status status.
+    - expired:
+        en: Canceled order (canceled: true) that does not have approved or pending payments (all rejected or returned).
+        es: Orden cancelada (cancelled:true) que no posee pagos aprobados ni pendientes (todos rechazados o devueltos)
```
Comment (lamaralmp) [inline reference/merchant_orders/resource.yaml:811]:
```suggestion
        es: Orden cancelada (cancelled: true) que no posee pagos aprobados ni pendientes (todos rechazados o devueltos)
```

**Pull request merged**